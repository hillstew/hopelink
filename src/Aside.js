import React from "react";

export default function Aside(props) {
  return (
    <aside>
      <p>{props.userName}</p>
      <button onClick={props.updateShowSavedAnswers}>See saved answers</button>
    </aside>
  );
}
