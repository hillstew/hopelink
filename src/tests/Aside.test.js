import React from "react";
import { shallow } from "enzyme";
import Aside from "../Aside";

let mockUserName = "Hillary"
let updateShowSavedAnswersMock = jest.fn();

describe('Aside', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Aside 
      userName={mockUserName}
      updateShowSavedAnswers={updateShowSavedAnswersMock}
    />)
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});