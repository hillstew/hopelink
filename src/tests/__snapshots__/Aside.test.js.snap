// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Aside should match snapshot 1`] = `
<aside>
  <p>
    Hillary
  </p>
  <button
    onClick={[MockFunction]}
  >
    See saved answers
  </button>
</aside>
`;
